scripts/k8s_deploy_dashboard_user.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# Make the dashboard a NodePort
+kubectl patch svc -n kube-system kubernetes-dashboard  -p '{"spec": {"type": "NodePort", "ports": [{"nodePort": 31443, "port": 443}] }}'
```

Comment:
Debated whether this should go here or live in a yaml file in config. I didn't necessarily want to add another config yaml for something that is not used very often and would have had to add more complex port discovery for url printing. This seemed simpler and good enough.